services/self-service/src/main/java/com/epam/dlab/backendapi/service/impl/ProjectServiceImpl.java

```diff
@@ -344,10 +344,11 @@ private String getUpdateBudgetAudit(ProjectDTO p) {
 		if (!configuration.isAuditEnabled()) {
 			return null;
 		}
-		Integer value = Optional.ofNullable(get(p.getName()).getBudget())
+		ProjectDTO dbProject = get(p.getName());
```

Comment:
Please rename **dbProject** to **project** or **projectDTO**

Reply:
Variable has been renamed